//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abc_O_A.h
//
// Code generation for function 'abc_O_A'
//

#ifndef ABC_O_A_H
#define ABC_O_A_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abc_O_A(double V_init, double A_init, double A_wayp, double V_min,
             double A_max, double J_max, double J_min, double t[7]);

#endif
// End of code generation (abc_O_A.h)
